docs/Deploying/Server-Installation.md

```diff
@@ -420,7 +420,75 @@ If you go back to [https://localhost:8080/hosts/manage](https://localhost:8080/h
 
 ## Deploying Fleet on Kubernetes
 
-In this guide, we're going to install Fleet and all of its application dependencies on a Kubernetes cluster. Kubernetes is a container orchestration tool that was open sourced by Google in 2014.
+In this guide, we're going to focus on deploying Fleet only on a Kubernetes cluster. Kubernetes is a container orchestration tool that was open sourced by Google in 2014.
+
+We will assume you have `kubectl` and MySQL and Redis are all set up and running. Optionally you have minikube to test your deployment locally on your machine.
+
+To deploy the Fleet server and connect to its dependencies(MySQL and Redis) we will set up a `deployment.yml` file with the following specifications:
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: fleet-deployment
+  labels:
+    app: fleet
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      app: fleet
+  template:
+    metadata:
+      labels:
+        app: fleet
+    spec:
+      containers:
+      - name: fleet
+        image: fleetdm/fleet
```

Comment:
This should specify a version which we then update with the current release of Fleet as time goes on. Otherwise it pulls "latest" which could be anything.

As part of a wider discussion, do we have a way to render variables like current release of Fleet into our docs?

Reply:
> As part of a wider discussion, do we have a way to render variables like current release of Fleet into our docs?

We don't. This would be a nice problem to solve, but is potentially tricky due to these variables being built into multiple "targets" (eg. fleetdm.com/docs and the [Helm charts](https://github.com/fleetdm/fleet/blob/main/.github/workflows/release-helm.yaml))

I agree that a version needs to be specified here. For now (until we solve the above problem), @DominusKelvin please update step 1 of https://fleetdm.com/docs/contributing/releasing-fleet to also point to where this needs to be changed for the Kubernetes documentation.

Reply:
Okay this means I should both update the image to point to a particular version of Fleet?

Reply:
Yes

Reply:
Added.